flake.nix

```diff
@@ -33,6 +32,8 @@
           })
           paths;
 
+        mapAttrs' = f: set:
+          builtins.listToAttrs (map (attr: f attr set.${attr}) (builtins.attrNames set));
```

Comment:
Nothing should be part of the public `fup` api, unless it adds serious `fup`-native value.

Reply:
Insofar, I guess I disagree with https://github.com/gytis-ivaskevicius/flake-utils-plus/pull/17#discussion_r609937344

Reply:
So I just added mapAttrs' to systemFlake. Its either we export it or keep it in the file. I see no reason to use mapAttrs' outside of this use, so I think its ok to keep it in systemFlake.